lib/engine/game/g_1849/game.rb

```diff
@@ -29,6 +32,11 @@ class Game < Game::Base
         BANK_CASH = 7760
 
         CERT_LIMIT = { 3 => 12, 4 => 11, 5 => 9 }.freeze
+        REDUCED_CERT_LIMIT = { 3 => 12, 4 => 9, 5 => 9 }.freeze
+
+        def game_cert_limit
+          reduced_4p_corps? ? self.class::REDUCED_CERT_LIMIT : self.class::CERT_LIMIT
```

Comment:
In this case, I'd avoid the duplication of 3 and 5 player values and instead do something like
```
limit = super.dup
limit[4] = 9 if reduced_4p_corps?
limit
```

Reply:
done.